[2024-07-28 11:55:03,989] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: send_stock_data.send_data scheduled__2024-07-28T11:54:00+00:00 [queued]>
[2024-07-28 11:55:03,998] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: send_stock_data.send_data scheduled__2024-07-28T11:54:00+00:00 [queued]>
[2024-07-28 11:55:03,999] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2024-07-28 11:55:03,999] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2024-07-28 11:55:03,999] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2024-07-28 11:55:04,013] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): send_data> on 2024-07-28 11:54:00+00:00
[2024-07-28 11:55:04,018] {standard_task_runner.py:52} INFO - Started process 636 to run task
[2024-07-28 11:55:04,020] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'send_stock_data', 'send_data', 'scheduled__2024-07-28T11:54:00+00:00', '--job-id', '25', '--raw', '--subdir', 'DAGS_FOLDER/***_stream.py', '--cfg-path', '/tmp/tmp9irk69ys', '--error-file', '/tmp/tmpis_0dhc9']
[2024-07-28 11:55:04,022] {standard_task_runner.py:80} INFO - Job 25: Subtask send_data
[2024-07-28 11:55:04,037] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.10/site-packages/***/configuration.py:528 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-07-28 11:55:04,084] {task_command.py:371} INFO - Running <TaskInstance: send_stock_data.send_data scheduled__2024-07-28T11:54:00+00:00 [running]> on host 77ea918c6e81
[2024-07-28 11:55:04,143] {taskinstance.py:1589} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=send_stock_data
AIRFLOW_CTX_TASK_ID=send_data
AIRFLOW_CTX_EXECUTION_DATE=2024-07-28T11:54:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-07-28T11:54:00+00:00
[2024-07-28 12:00:05,180] {logging_mixin.py:115} INFO - Gửi thất bại: KafkaError{code=_MSG_TIMED_OUT,val=-192,str="Local: Message timed out"}
[2024-07-28 12:00:05,181] {logging_mixin.py:115} INFO - Sent: {'date_time': '26/07/2024, 20:00:01', 'open': 218.85, 'high': 219.49, 'low': 216.01, 'current_price': 217.96, 'previous_close': 217.49, 'change_percent': 0.2161}
[2024-07-28 12:00:07,404] {python.py:173} INFO - Done. Returned value was: None
[2024-07-28 12:00:07,418] {taskinstance.py:1415} INFO - Marking task as SUCCESS. dag_id=send_stock_data, task_id=send_data, execution_date=20240728T115400, start_date=20240728T115503, end_date=20240728T120007
[2024-07-28 12:00:07,439] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-07-28 12:00:07,462] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
